index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Twitter Layout Challenge DN</title>
+    <title>Twitter Layout Challenge - DN</title>
```

Comment:
You don't have many commits.  This makes reviewing harder as lots of changes done in each commit. Also think the commit comments could be more detailed to help know exact changes.